### PR TITLE
Fix/drivers age

### LINKED ISSUE #112: bloquer l'âge du conducteur à deux chiffres
### Comportement actuel
pas de limites

### Comportement attendu
blocquer l'âge à des valeurs à 2 chiffres (Axel)

### Étapes pour reproduire le problème

[] Développement
[] Production

### PR DESCRIPTION
**Pull request by webplate**
### What does/resolve this PR? 
Resolve #112 
Add some type checking and min / max limits

### CODE REVIEW

File: app/models/trip.rb [hunk 1]
```diff
@@ -18,9 +18,9 @@ class Trip < ApplicationRecord
   validates_inclusion_of :comfort, in: CAR_RATINGS
   validates_inclusion_of :state, in: STATES
   validates_inclusion_of :departure_date, in: Time.zone.today..Time.zone.today+1.year, message: "Mettre une date situé entre aujourd hui et dans 1 an."
-  validates_numericality_of :seats, { greater_than_or_equal_to: 1 }
-  validates_numericality_of :price, { greater_than_or_equal_to: 0 }
-  validates_numericality_of :age, allow_blank: true
+  validates_numericality_of :seats, only_integer: true, greater_than_or_equal_to: 1
+  validates_numericality_of :price, greater_than_or_equal_to: 0
+  validates_numericality_of :age, only_integer: true, allow_blank: true, greater_than: 0, less_than: 100
```
Comment (nicolasleger) [inline app/models/trip.rb]: Only euros no cents ?

Comment (webplate) [reply app/models/trip.rb]: its the age so not a float

Comment (webplate) [reply app/models/trip.rb]: i fixed it :)

Comment (maxb29) [reply app/models/trip.rb]: Could we add the "0" for the nb of seats?
cf : https://github.com/covoiturage-libre/covoiturage-libre/issues/141

Comment (webplate) [reply app/models/trip.rb]: Good proposition, but I won't do it here. I prefer to keep PRs synthetic, this is out of the scope.
Maybe in another one ?

Comment (Poussinou) [reply app/models/trip.rb]: @webplate Pourquoi ne pas mettre un âge supérieur à 18 par ailleurs?

**Pull request merged**